account_invoice_line_stock_move_info/__openerp__.py

```diff
@@ -0,0 +1,33 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Account Invoice Line Stock Move Info',
+    'version': "1.0",
+    'author': 'OdooMRP team',
```

Comment:
Ayer lo vimos y la mayoría de módulos que tenéis están como AvanzOSC, así que lo he puesto así. Por cierto, ya están los módulos en apps: https://www.odoo.com/apps/modules?author=OdooMRP%20team